drivers/gpio/gpio_nrfx.c

```diff
@@ -513,7 +515,7 @@ static int gpio_nrfx_init(struct device *port)
 #define GPIO_NRF_DEVICE(id)						\
 	static const struct gpio_nrfx_cfg gpio_nrfx_p##id##_cfg = {	\
 		.common = {						\
-			.port_pin_mask = GPIO_PORT_PIN_MASK_FROM_NGPIOS(DT_INST_##id##_NORDIC_NRF_GPIO_NGPIOS), \
+			.port_pin_mask = GPIO_PORT_PIN_MASK_FROM_NGPIOS(DT_INST_PROP(id, ngpios)), \
```

Comment:
wonder if you can do a line wrap here so as not to exceed 80 characters

Reply:
It didn't line wrap before.  (this was done via a script, so dealing with whitespace / line wrap is a bit of a pain).